packages/psss/src/api/queries/useSingleWeeklyReport.js

```diff
@@ -6,6 +6,14 @@
 import { queryCache } from 'react-query';
 import { useTableData } from './useTableData';
 
+const SYNDROMES = {
+  AFR: 'Acute Fever and Rash (AFR)',
```

Comment:
I used uppercase ids assuming that they are arbitrary. This allows us to more easily construct keys for data returned from the existing reports